QuantLib/ql/termstructures/volatility/optionlet/optionletstripper.hpp

```diff
@@ -40,6 +40,7 @@ namespace QuantLib {
       public:
         //! \name StrippedOptionletBase interface
         //@{
+        enum Model{Normal, ShiftedLognormal};
```

Comment:
Yes, shifted as in "using a displacement". 
Your second question. Well, the idea has been to keep signatures of normal and (displaced) lognormal engines (Bachelier vs Black enginge) identical. The displacement is actually not used in the normal case. By doing this one could possibly try a templated approach for the optionletstripper instead of the rather naive *if model is a then ...* approach. Is this clear? I was just not sure, whether the templated version is worth the hassle of changing "the rest" of the existing code. Therefore I decided to first go for the simple *if / else* approach to see if there is any interest in the contribution.